src/source.rs

```diff
@@ -71,8 +71,12 @@ impl Default for CallbackGuard {
 
 impl Drop for CallbackGuard {
     fn drop(&mut self) {
+        use std::io::stderr;
+        use std::io::Write;
+
         if thread::panicking() {
-            process::exit(101);
+            let _ = stderr().write(b"Uncaught panic, exiting");
```

Comment:
Anything more useful to print here, anything to add?

Reply:
IMHO nothing.

Reply:
Actually something. It needs a \n at the end. @GuillaumeGomez , can you just push that simple change? Going through a PR seems a bit too much overhead for one character :)

Reply:
Ah sure.